src/vrf03.rs

```diff
@@ -550,4 +550,52 @@ mod test {
             assert_eq!(output[..], hex::decode(vector[3]).unwrap());
         }
     }
+
+    use std::{fs::File, path::Path};
+
+    #[test]
+    fn check_against_cardano_base() {
+        check_against_golden("./tests/test_vectors/vrf_ver03_generated_1");
```

Comment:
create a static `const test_vectors : [&'static str; 8] = [ .... ]` and then iterate over the vector, eg. a parametric test

Reply:
done